### DISCUSSION TITLE
publish to jsr.io?

### DISCUSSION THREAD

**Discussion by iuioiua** [panva/jose#695] [Ideas]
I'd be happy to help.

**Comment by panva** [comment]
When it (jsr) is more stable and established. So far I am not convinced by the available tooling and my early attempts to use jsr were in vain. There are plenty distribution channels to choose from for all applicable runtimes.

**Reply by iuioiua** [reply]
Are you able to elaborate on what you mean by available tooling?

**Reply by panva** [reply]
I mean jsr in general. Last time I checked, the CLI used to publish and the public jsr documentation for publishing modules were not aligned, and there were issues in the jsr repo that needed to be resolved first before it was even possible to use.

I personally don't have high hopes for JSR and don't believe that the registry should be responsible for taking typescript as input and giving out transpiled js as output to runtimes that don't support TS directly.

Happy to be proven otherwise as time goes on. As is figuring out how to add non-1.0 jsr.io to the publish steps is in my opinion a waste of time.
